function.tf

```
@@ -0,0 +1,78 @@
+locals {
+  enable_function = var.enable_function || var.enable_extensions
+}
+
+resource "google_service_account" "cloud_functions" {
+  account_id  = "${var.name}-func"
+  description = "A service account for the Observe Cloud Functions"
```

```suggestion
  description = "Used by the Observe Cloud Function"
```

Avoid repeating the resource type in the name/description